### PR TITLE
fix(tabs): add component tokens

### PR DESCRIPTION
**Pull request by Elijbet**
**Related Issue:** #7180

## Summary

Adds the following component tokens:
```
--calcite-tabs-background-color
--calcite-tabs-border-color 
 ```

### CODE REVIEW

File: packages/calcite-components/src/components/tabs/tabs.scss [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+ * @prop --calcite-tabs-background-color: The background color of the component.
+ * @prop --calcite-tabs-border-color: The border color of the component.
+ */
+
 :host {
   @apply flex flex-col;
 }
 
 :host([bordered]) {
+  --calcite-tabs-background-color: var(--calcite-color-foreground-1);
```
Comment (driskull) [inline packages/calcite-components/src/components/tabs/tabs.scss:15]: I think these should be on the host and not `:host([bordered]` right?

**Review by driskull — APPROVED** [review]
Looks good once vars are moved to host.

File: packages/calcite-components/src/components/tabs/tabs.scss (hunk 1 above)
Comment (Elijbet) [reply packages/calcite-components/src/components/tabs/tabs.scss:15]: I think background color is only set on bordered.

Comment (Elijbet) [reply packages/calcite-components/src/components/tabs/tabs.scss:15]: That's right, made a blunder.

**Review by jcfranco — APPROVED** [review]

**Pull request merged**